kaspad.go

```diff
@@ -71,13 +81,19 @@ func newKaspad(listenAddrs []string, interrupt <-chan struct{}) (*kaspad, error)
 
 	txMempool := setupMempool(dag, sigCache)
 
+	protocolManager, err := protocol.NewProtocol(listenAddrs, dag)
```

Comment:
I'd prefer if you just call it "protocol". If your linter is upset about it hiding the package, just call it p.